### PR TITLE
Renamed the one_off_loader to uncached_loader, and sets saleArtworkRootLoader to be one

### PR DESCRIPTION
**Pull request by orta**
1. Makes the loader look like the rest of the loaders as a consumer
2. Changes `saleArtworkRootLoader` to be uncached
3. WIP because I want to sit down with you and make sure it's 100% - the system time API still works

### CODE REVIEW

**Comment by sweir27** [comment]
Cool!! Yeah let's take a look at this tomorrow 👍

**Comment by orta** [comment]
We verified this worked as expected

File: src/lib/loaders/api/loader_without_cache_factory.ts [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
-  options: any
-) => {
+  _apiName: string
+) => (path: string | FuncToString, options: any | null) => {
   // If you use gravity as the api here, then options will get interpreted as
   // an accessToken, so you have to explicitly pass null
   const loader = new DataLoader(
-    () => Promise.resolve([api(path, options).then(r => r.body)]),
+    ([route]) =>
+      Promise.resolve([api(route as string, options).then(r => r.body)]),
```
Comment (alloy) [inline src/lib/loaders/api/loader_without_cache_factory.ts]: How about typing like `new DataLoader<string, string>(…)` instead of casting here?

**Comment by alloy** [comment]
LGTM 👍 Needs an update, though.

**Comment by orta** [comment]
Cool, done - merge on green

**Pull request merged**